Update frontpage pitch (#44)
r-multiverse/r-multiverse.github.io: index.md

<br>
<br>

> R-multiverse is a new initiative to establish a trustworthy, community-curated R packages repository that runs on the open-source [r-universe](https://r-universe.dev) infrastructure. We strive for [transparent governance](policies.md), high automatation, and quarterly production releases. Visit the [overview page](overview.qmd) page to learn more.

Reviewer: ```suggestion
R-multiverse is a new initiative to establish a trustworthy, community-curated R packages repository that runs on the open-source [r-universe](https://r-universe.dev) infrastructure. 
We strive for [transparent governance](policies.md), high automatation, and quarterly production releases. 
Visit the [overview page](overview.qmd) page to learn more.
```

> R-multiverse is a new initiative to establish a democratic, multi-stakeholder, and trustworthy R packages repository that runs on the open-source [r-universe](https://r-universe.dev) infrastructure. We strive for [transparent governance](policies.md), high automatation, and quarterly production releases. Visit the [overview page](overview.qmd) page to learn more.

Reviewer 2: ```suggestion
R-multiverse is a new initiative to establish a trustworthy, community-curated R packages repository that runs on the open-source [r-universe](https://r-universe.dev) infrastructure. We strive for [transparent governance](policies.md), high automatation, and quarterly production releases. Visit the [overview page](overview.qmd) page to learn more.
```

Reviewer 2: I preferred your original snappier version 😄

Author: I thought it would be good to somehow emphasize the multi stands for multiple organizations or stakeholders are involved (to contrast with e.g. cran). But you prefer community-curated?

Reviewer 2: I do like to have 'community' in there somewhere - that lends it legitimacy as well as just being 'nice'.

<br>
<br>

> R-multiverse is a new initiative to establish a trustworthy, community-curated R packages repository that runs on the open-source [r-universe](https://r-universe.dev) infrastructure. 

Reviewer 3: ```suggestion
R-multiverse is a new initiative to establish a trustworthy, community-curated R package repository that runs on the open-source [r-universe](https://r-universe.dev) infrastructure.
```
